### ISSUE TITLE
experimental secondary auxiliary heater when boost called

### ISSUE THREAD

**Issue by feelingwalnut** [999LV/SmartVirtualThermostat#68]
@999LV I;m not sure if this should be a pull or is even in line with the intent of your boost function.
https://github.com/feelingwalnut/SmartVirtualThermostat/tree/secondary-boost(aux)-heater

it includes the temperature C-F identifying and conversion.

**Comment by 999LV** [comment]
Hi, interesting but probably very userspecific indeed.

**Comment by feelingwalnut** [comment]
yes very specific use case. in this climate it is common for homes to have multiple sources of heat, often one far more expensive to use than the other.
such as a mix of any two: gas, pellets, coal, wood(there are models that can automate) electric or oil.
you typically would run the cheaper but frequently in winters it gets cold enough to overrun your main heat, causing the second to need used.

maybe your home would have an electric heat pump, but when it is cold enough that switches to an internal electric resistance heater. you'd then want your electric baseboards to turn on as well.
maybe you use pellets but when cold enough you'd want your gas to turn on in addition.
